lua/cinema/theater/services/sh_bflix.lua

```diff
@@ -81,27 +83,34 @@ if CLIENT then
                     end
                 end)
 
+                -- The main video page
                 if url == key then
+                    self:QueueJavascript([[
+                        const title = document.querySelector('body div#aa-wp div.bd.cont section.section.single div.rw aside.left.cl1 article.post.single div.dfxb div header.entry-header h1.entry-title').textContent;
```

Comment:
this is way too specific. will almost certainly break

Reply:
done